src/routes/user_with_google.route.js

```diff
@@ -28,6 +28,6 @@ userRouter.get(
   })
 );
 
-userRouter.get('/auth/google/success', userControllers.loginWithGoogle);
+userRouter.post('/auth/google/success', userControllers.loginWithGoogle);
```

Comment:
does this requires to change the swagger documentation?

Reply:
no

Reply:
no